include/deal.II/grid/grid_generator.h

```diff
@@ -548,15 +548,25 @@ namespace GridGenerator
    * both the faces and the edges of these boundaries. If the flag is @p
    * false, both have indicator zero.
    *
+   * Attach a SphericalManifold for correct placement of vertices upon refinement
+   * and higher order mappings. Alternatively, attach a HyperShellBoundary to the
+   * inner and outer boundary.
+   *
    * In 2d, the number <tt>n_cells</tt> of elements for this initial
    * triangulation can be chosen arbitrarily. If the number of initial cells
    * is zero (as is the default), then it is computed adaptively such that the
    * resulting elements have the least aspect ratio.
    *
-   * In 3d, only certain numbers are allowed, 6 for a surface based on a
+   * In 3d, only certain numbers are allowed, 6 (or the default 0) for a surface based on a
    * hexahedron (i.e. 6 panels on the inner sphere extruded in radial
    * direction to form 6 cells), 12 for the rhombic dodecahedron, and 96 (see
-   * below). These give rise to the following meshes upon one refinement:
+   * below).
+   *
+   * While the SphericalManifold, that is demonstrated in the documentation of the
+   * @ref manifold "documentation module on manifolds", creates reasonable meshes
+   * for any number of @p n_cells, the situation is less than ideal when only attaching
+   * a HyperShellBoundary. Then, the 3d meshes give rise to the following meshes
+   * upon one refinement:
```

Comment:
It took me a little while to understand the distinction. Can you say somehow that the former is a manifold description for *all cells* whereas the latter is a boundary description only for those faces and edges on the boundary?